scripts/utils.ts

```diff
@@ -276,6 +317,65 @@ export function getBeefyBnbVenusWithdrawOrder(context: FactoryAndOperatorsForkin
     ];
 }
 
+// Create a Deposit order in StakeDAO
+export function getStakeDaoDepositETHOrder(context: FactoryAndOperatorsForkingETHFixture, strategyAddress: string, amountToDeposit: BigNumber, minStrategyToken?: BigNumber) {
+    return [
+        buildOrderStruct(context.stakeDaoCurveStrategyDepositETHOperatorNameBytes32, strategyAddress, [
+            ["address", strategyAddress],
+            ["uint256", amountToDeposit],
+            ["uint256", minStrategyToken != null ? minStrategyToken : 0], // 100% slippage if minAmountOut is null
+        ]),
+    ];
+}
+
+// Create a Deposit order in StakeDAO
+export function getStakeDaoDepositOrder(context: FactoryAndOperatorsForkingBSCFixture, strategyAddress: string, tokenToDeposit: string, amountToDeposit: BigNumber, minStrategyToken?: BigNumber) {
+    return [
+        buildOrderStruct(context.stakeDaoCurveStrategyDepositOperatorNameBytes32, context.stakeDaoUsdStrategyAddress, [
+            ["address", strategyAddress],
+            ["address", tokenToDeposit],
+            ["uint256", amountToDeposit],
+            ["uint256", minStrategyToken != null ? minStrategyToken : 0], // 100% slippage if minAmountOut is null
+        ]),
+    ];
+}
+
+// Create a Withdraw128 order in StakeDAO
+export function getStakeDaoWithdrawETHOrder(context: FactoryAndOperatorsForkingETHFixture, strategyAddress: string, amountToWithdraw: BigNumber, minAmountOut?: BigNumber) {
+    return [
+        buildOrderStruct(context.stakeDaoCurveStrategyWithdrawETHOperatorNameBytes32, strategyAddress, [
+            ["address", strategyAddress],
+            ["uint256", amountToWithdraw],
+            ["uint256", minAmountOut != null ? minAmountOut : 0], // 100% slippage if minAmountOut is null
+        ]),
+    ];
+}
+
+// Create a Withdraw128 order in StakeDAO
+export function getStakeDaoWithdraw128Order(context: FactoryAndOperatorsForkingBSCFixture, strategyAddress: string, amountToWithdraw: BigNumber, outputToken: string, minAmountOut?: BigNumber) {
+    return [
+        buildOrderStruct(context.stakeDaoCurveStrategyWithdraw128OperatorNameBytes32, context.stakeDaoUsdStrategyAddress, [
```

Comment:
I think we can use `strategyAddress` instead of the context

Reply:
Yes we can use `strategyAddress`.

We also need to use a real StakeDAO strategy address (but not whitlisted) in the test `Should revert if strategy is not whitelisted` (under `withdraw128()` describe) to keep the test pass.